charts/hivemq-platform/templates/hivemq-custom-resource.yml

```diff
@@ -58,7 +58,7 @@ spec:
                 {{- $envNamesKeyStoreList := list }}
                 {{- $envNamesTrustStoreList := list }}
                 {{- range $key, $val := .Values.services }}
-                {{- if and $val.exposed (or $val.keystorePassword $val.keystorePasswordSecretName) }}
+                {{- if and $val.exposed $val.keystoreSecretName }}
```

Comment:
A few lines below we only have `{{- if $val.truststoreSecretName }}`, without the exposed. Do we need to align these two?